src/SSHDebugPS/IConnection.cs

```diff
@@ -121,15 +123,37 @@ public Process(uint id, string arch, uint flags, string userName, string command
         }
     }
 
+    internal static class OperatingSystemStringConverter
+    {
+        internal static PlatformID ConvertToPlatformID(this string value)
```

Comment:
>this [](http://example.com/codeflow?start=55&length=4)

Nit: why make it an extension method on `string` instead of just a normal static method?

Reply:
I was being too fancy, but after thinking about it since its not going to be a common API for string, its just make it a non-extension method.